source/_patterns/01-molecules/item-details/item-details.twig

```diff
@@ -0,0 +1,39 @@
+{%- with list only -%}
+  {%- block list -%}
```

Comment:
Since the whole thing is just a list, we can get rid of `list` (both the block and the variable).